src/unity/toolkits/supervised_learning/supervised_learning.hpp

```diff
@@ -750,7 +750,10 @@ class EXPORT supervised_learning_model_base : public ml_model_base {
 
   register_defaults("evaluate",
                     {{"metric", std::string("_report")},
-                     {"missing_value_action", std::string("auto")}});
+                     {"missing_value_action", std::string("auto")},
+                     {"predictions", gl_sarray()}, 
+                     {"with_predictions", true}
```

Comment:
This will change the default when called from Python to `with_predictions=True`, right? (Or does Python explicitly pass that in?)

Reply:
Ah, good catch.  I think python almost always passes it in explicitly, but we might as well set this to false.